plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.ts

```diff
@@ -229,10 +229,11 @@ export function createFetchTemplateAction(options: {
           await fs.ensureDir(outputPath);
         } else {
           const inputFilePath = resolveSafeChildPath(templateDir, location);
+          const stats = await fs.promises.lstat(inputFilePath);
 
-          if (await isBinaryFile(inputFilePath)) {
+          if (stats.isSymbolicLink() || (await isBinaryFile(inputFilePath))) {
```

Comment:
I'm thinking we add a similar check to `serializeDirectoryContents` but in that case we just ignore the file instead for now.